sample-app/src/main/java/com/joinforage/android/example/ui/pos/POSComposeApp.kt

```diff
@@ -68,6 +71,11 @@ fun POSComposeApp(
         mutableStateOf(null)
     }
 
+    val context = LocalContext.current
+    val k9SDK by remember {
```

Comment:
`remember` will persist across "recompositions" (re-renders in React-speak) but `rememberSaveable` will persist across recompositions _and_ configuration changes. I'm don't think it matters either way for this, just calling it out in case it does.

Reply:
I tried to use `rememberSaveable` and the app crashed. I thiiiiiink it had to do with the fact that `K9SDK` is not serializable. I didn't bother to investigate a workaround at the time but I completely agree that `rememberSaveable` is better. I'll make a linear ticket to track this tech debt because I bet this will cause a bug at some point in the future. Great callout!

Reply:
Ahh I got ya, makes sense. Tbh I'm not sure this app will undergo many configuration changes anyway 😅